src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -495,6 +495,11 @@ export default class DefaultAudioVideoController
           this.configuration.credentials.attendeeId
         );
       }
+      if (this.meetingSessionContext.videoUplinkBandwidthPolicy.setTransceiverController) {
+        this.meetingSessionContext.videoUplinkBandwidthPolicy.setTransceiverController(
+          this.meetingSessionContext.transceiverController
+        );
+      }
       this.meetingSessionContext.audioProfile = this._audioProfile;
     }
```

Comment:
Just set a boolean field here.

Do you need to also check unified plan here?

Reply:
Updated.